python/Ganga/test/GPI/TestSJSubmit.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+class TestSJSubmit(GangaUnitTest):
+
+    n_subjobs = 10
+
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSJSubmit, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('Output', 'FailJobIfNoOutputMatched', 'True')
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
Yes I am relying on this happening to test different sections of code. The behaviour changes between loading a job from disk and running with it in memory. Arguably I should add a job.resubmit whilst in the same session but this is tested elsewhere.